src/databricks/labs/ucx/workspace_access/scim.py

```diff
@@ -12,6 +13,7 @@
     Crawler,
     Destination,
     Permissions,
+    logger,
```

Comment:
Create logger in this module, don't import it from other modules, as we'll have wrong error messages